packages/orchestrator-ui-components/src/utils/date.ts

```diff
@@ -9,6 +9,35 @@ export const parseDate = (date: string | null | undefined): Date | null => {
     return new Date(date);
 };
 
+export const calculateTimeDifference = (from: string, to: string): string => {
+    const fromDate = parseDate(from);
+    const toDate = parseDate(to);
+
+    if (!fromDate || !toDate) return 'missing paramater';
```

Comment:
According to the parameter types this situation is not possible. Else the type would be string | undefined.
About the handling I would suggest to not return an error message as a string, but rather return an empty string or "0:00:00". Similar idea the "negative difference" return below

Reply:
This supresses typescript errors in the code below that come from parseDate possibly returning Null so I do prefer leaving it in. And who nows, it might get triggered somewhere in the future somehow

I've considered returning 0.00.00 but that feels incorrect since it's not true. I've also considered throwing for both cases where I return a string now but that creates the need to handle the exception. This way at least you see the error very clearly in frontend code calling it. Toughts?

Reply:
parseDate possibly null, check! And my thought on returning values, I would go for empty string or "?". These return values seem more debug info. Maybe do check the requirements of these corner cases with Wouter/Hans

Reply:
ok. changed it to empty string